source/frontend/public/manifest.json

```diff
@@ -7,7 +7,7 @@
       "sizes": "17x17"
     },
     {
-      "src": "tenants/moti/PIMS-logo.png",
+      "src": "tenants/MOTI/PIMS-logo.png",
```

Comment:
not sure this file is used at all. AFAIK the manifest is only needed if the app is configured to be used as a PWA